tests/Unit/ApparentHorizons/Test_FastFlow.cpp

```diff
@@ -0,0 +1,283 @@
+        gr::extrinsic_curvature(
+            get<gr::Tags::Lapse<3, Frame::Inertial, DataVector>>(vars),
+            get<gr::Tags::Shift<3, Frame::Inertial, DataVector>>(vars),
+            get<EinsteinSolutions::KerrSchild::deriv_shift<DataVector>>(vars),
+            spatial_metric,
+            get<gr::Tags::DtSpatialMetric<3, Frame::Inertial, DataVector>>(
+                vars),
+            deriv_spatial_metric),
+        raise_or_lower_first_index(
+            compute_christoffel_first_kind(deriv_spatial_metric),
+            inverse_spatial_metric));
+    status = status_and_info.first;
+  }
+  return status;
+}
+
+struct FastFlowFromOpts {
+  using type = FastFlow;
+  static constexpr OptionString help{"FastFlow horizon-finder"};
+};
+
+void test_construct_from_options_fast() {
+  Options<tmpl::list<FastFlowFromOpts>> opts("");
+  opts.parse(
+      "FastFlowFromOpts:\n"
+      "  Flow: Fast\n"
+      "  Alpha: 1.1\n"
+      "  Beta: 0.6\n"
+      "  AbsTol: 1.e-10\n"
+      "  TruncationTol: 1.e-3\n"
+      "  DivergenceTol: 1.1\n"
+      "  DivergenceIter: 6\n"
+      "  MaxIts: 200");
+  CHECK(opts.get<FastFlowFromOpts>() ==
+        FastFlow(FastFlow::FlowType::Fast, 1.1, 0.6, 1e-10, 1e-3, 1.1, 6, 200));
+}
+
+void test_construct_from_options_jacoby() {
+  Options<tmpl::list<FastFlowFromOpts>> opts("");
+  opts.parse(
+      "FastFlowFromOpts:\n"
+      "  Flow: Jacoby\n"
+      "  Alpha: 1.1\n"
+      "  Beta: 0.6\n"
+      "  AbsTol: 1.e-10\n"
+      "  TruncationTol: 1.e-3\n"
+      "  DivergenceTol: 1.1\n"
+      "  DivergenceIter: 6\n"
+      "  MaxIts: 200");
+  CHECK(opts.get<FastFlowFromOpts>() == FastFlow(FastFlow::FlowType::Jacoby,
+                                                 1.1, 0.6, 1e-10, 1e-3, 1.1, 6,
+                                                 200));
+}
+
+void test_construct_from_options_curvature() {
+  Options<tmpl::list<FastFlowFromOpts>> opts("");
+  opts.parse(
+      "FastFlowFromOpts:\n"
+      "  Flow: Curvature\n"
+      "  Alpha: 1.1\n"
+      "  Beta: 0.6\n"
+      "  AbsTol: 1.e-10\n"
+      "  TruncationTol: 1.e-3\n"
+      "  DivergenceTol: 1.1\n"
+      "  DivergenceIter: 6\n"
+      "  MaxIts: 200");
+  CHECK(opts.get<FastFlowFromOpts>() == FastFlow(FastFlow::FlowType::Curvature,
+                                                 1.1, 0.6, 1e-10, 1e-3, 1.1, 6,
+                                                 200));
+}
+
+void test_serialize() noexcept {
+  FastFlow fastflow(FastFlow::FlowType::Jacoby, 1.1, 0.6, 1e-10, 1e-3, 1.1, 6,
+                    200);
+  test_serialization(fastflow);
+}
+
+void test_copy_and_move() noexcept {
+  FastFlow fastflow(FastFlow::FlowType::Curvature, 1.1, 0.6, 1e-10, 1e-3, 1.1,
+                    6, 200);
+  test_copy_semantics(fastflow);
+  auto fastflow_copy = fastflow;
+  // clang-tidy: std::move of triviable-copyable type
+  test_move_semantics(std::move(fastflow), fastflow_copy);  // NOLINT
+}
+
+void test_ostream() noexcept {
```

Comment:
use get_output

Reply:
Do you mean rename this function to get_output, or is there some other get_output function I should use?

Reply:
use function in TestHelpers